internal-convert.py

```diff
@@ -3,10 +3,11 @@
 from urlparse import urlparse
 import sys
 import yaml
+from future.utils import iteritems
 
-distribution = yaml.load(sys.stdin)
+distribution = yaml.unsafe_load(sys.stdin)
 
-for repository_name, repository in distribution['repositories'].iteritems():
+for repository_name, repository in iteritems(distribution['repositories']):
```

Comment:
Why `iteritems` vs just calling `.items`?

Reply:
See https://www.python.org/dev/peps/pep-0469/

Specifically: "Migrating to the common subset of Python 2 and 3"

The code from future.utils implements the right if-then statements to ensure that this code works on both Python 2.6+ and Python 3.

Reply:
(This is necessary since we are still running Github Action checks on both Bionic [Python 2] and Focal [Python 3])

Reply:
Correct me if I'm wrong, but I don't think the iterator aspect of it is actually important in this refactor (unless it's being changed during iteration...), so just calling `.items` seems fine to me. That method is supported in both Python 2 and 3.